### PR TITLE
feat(TDP-1452): updated live blog schema to have live blog updates

### PR DESCRIPTION
**Pull request by adamosborne-tnl**
I have the **LiveBlogPosting** schema to include the incremental **liveBlogUpdate** values in the schema. This is mostly the ArticleHeader divider data provided by these headers 
image
There is also additional information provided by the subsequent paragraphs, images or videos included in the update.

you should be able to validate the schema at https://validator.schema.org/ by looking for the **liveBlogUpdate** data
image

### CODE REVIEW

File: packages/ts-components/src/components/article-header/ArticleHeader.tsx [hunk 1]
```diff
@@ -38,13 +38,13 @@ const ArticleHeader: React.FC<{
   const isLessThan1Hour = diffInSeconds < 60 * 60;
   const isLessThan13Hours = diffInSeconds < 60 * 60 * 13;
   const isDaysAgo = differenceInCalendarDays(currentDateTime, updatedDate) >= 1;
-
+  const anchorPoint = `u_${window.btoa(updated)}`;
```
Comment (flashcheeks) [inline packages/ts-components/src/components/article-header/ArticleHeader.tsx:41]: `updated` might not be unique. maybe we can add the index value?

File: packages/article-skeleton/src/head.js [hunk 2]
```diff
@@ -130,6 +130,62 @@ const getThumbnailUrl = article => {
   return crop ? crop.url : "";
 };
 
+const getLiveBlogUpdates = (article, publisher, author) => {
+  const updates = [];
+  let update;
+
+  for (let i = 0; i < article.content.length; i += 1) {
```
Comment (flashcheeks) [inline packages/article-skeleton/src/head.js:137]: `article.content` also contains a `paywall` object that you need to parse, or has this been normalised already?

**Review by flashcheeks — CHANGES_REQUESTED** [review]

File: packages/ts-components/src/components/article-header/ArticleHeader.tsx (hunk 1 above)
Comment (adamosborne-tnl) [reply packages/ts-components/src/components/article-header/ArticleHeader.tsx:41]: Only needs to be unique enough for the page and I somewhat doubt that they would have multiple headlines to the second on the same blog page. I use the same value in the head.js for the schema so didn't want to curry the value that may not be the same elsewhere

File: packages/article-skeleton/src/head.js (hunk 2 above)
Comment (adamosborne-tnl) [reply packages/article-skeleton/src/head.js]: Hmm, only working on storybook, got any ideas on how i could check that?

Comment (adamosborne-tnl) [reply packages/article-skeleton/src/head.js]: would i need to check though `content` and `paywalledContent` ?

Comment (adamosborne-tnl) [reply packages/article-skeleton/src/head.js]: ok, updated for `paywall`

File: packages/ts-components/src/components/article-header/ArticleHeader.tsx (hunk 1 above)
Comment (DMackintosh-TNL) [reply packages/ts-components/src/components/article-header/ArticleHeader.tsx]: Should it be back ticked?

Comment (adamosborne-tnl) [reply packages/ts-components/src/components/article-header/ArticleHeader.tsx]: yes, but that's an old line now.

**Review by DMackintosh-TNL — APPROVED** [review]
Looks good mate :)

**Review by flashcheeks — APPROVED** [review]

**Pull request merged**